tests/Integration/Momento.Sdk.Tests/SimpleCacheDataTest.cs

```diff
@@ -38,14 +38,20 @@ public async Task SetAsync_KeyIsByteArrayValueIsByteArray_HappyPath()
         byte[] key = Utils.NewGuidByteArray();
         byte[] value = Utils.NewGuidByteArray();
         await client.SetAsync(cacheName, key, value);
-        byte[]? setValue = (await client.GetAsync(cacheName, key)).ByteArray;
-        Assert.Equal(value, setValue);
+        CacheGetResponse response = await client.GetAsync(cacheName, key);
+        if (response is CacheGetResponse.Success goodResponse) {
+            byte[]? setValue = goodResponse.ByteArray;
+            Assert.Equal(value, setValue);
+        }
```

Comment:
Test should fail if response is not a success. The previous test would have failed if not